### PR TITLE
Note page layout development

### PR DESCRIPTION
**Pull request by dev-enforced**
In this PR one can find the screen styling for the Note IT app which the user will see once he logs into the app
So first you need to click on Make Notes that will take you to the login page there you can either login using these credentials:
```js
email address: adarshbalika@gmail.com
password: adarshBalika123
```
or use the guest sign in functionality that will take you to the notes app main screen where you can see a navbar and sidebar having the possible routes of All Notes,Archives,Labels and Trash Items
Here is the live preview link for the same: https://deploy-preview-4--note-simply.netlify.app/noteshome

### CODE REVIEW

File: src/constants/icons.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { HiOutlineMenu as Menu } from "react-icons/hi";
+import { RiDeleteBin7Line as Delete, RiPencilLine as Pen } from "react-icons/ri";
+import { MdOutlineLabel as Label, MdOutlinePushPin as PinOutline, MdPushPin as PinFilled } from "react-icons/md";
+import { BiNotepad as Notes, BiArchiveIn as Archived } from "react-icons/bi";
+import { IoColorPaletteOutline as ColorPalette } from "react-icons/io5"
+export {Menu,Delete,Label,Notes,Archived,PinOutline,PinFilled,ColorPalette,Pen}
```
Comment (mariyasada) [inline src/constants/icons.js:6]: liked that you have created separate file for icons

**Review by mariyasada — APPROVED** [review]
this code looks fine to me. good to merge

File: src/components/SideNavigation/SideNavigation.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    max-width:12rem;
+}
+
+.side-nav-link{
+    color:var(--primary-color);
+    font-size: 1.4rem;
+    border-radius: var(--border-rd-sm);
+    width: fit-content;
+    margin: 1.2rem 0;
+    padding:0.2rem 0.8rem;
```
Comment (ShreyashMahajan) [inline src/components/SideNavigation/SideNavigation.css:11]: px for less than 1rem

File: src/constants/sideNavigationLinks.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { Delete, Label, Notes, Archived } from 'constants';
+const sideNavigationLinks = [
```
Comment (ShreyashMahajan) [inline src/constants/sideNavigationLinks.js:2]: i liked you created separated array for links

**Review by ShreyashMahajan — APPROVED** [review]
UI is amazing, simple, and decent. code structured is also very good.

File: src/components/SideNavigation/SideNavigation.css (hunk 2 above)
Comment (dev-enforced) [reply src/components/SideNavigation/SideNavigation.css:11]: Yeah just for consistency kept it in this manner

**Pull request merged**